### PR TITLE
Add documentation for possible segfault in object.h

### PR DESCRIPTION
**Pull request by xVin**
Addresses #89, added documentation.

### CODE REVIEW

**Comment by neithernut** [comment]
Please add newlines to seperate the warning from the rest of the documentation.
The text itself is ok.
:+1:

**Comment by xVin** [comment]
Update.

File: src/objects/object.h [hunk 1]
```diff
@@ -237,7 +237,11 @@ ws_object_getref(
  *
  * @memberof ws_object
  *
- * @warning It is not save to use the object after this operation _in any kind_.
```
Comment (matthiasbeyer) [inline src/objects/object.h:240]: This shouldn't be removed.

Comment (xVin) [inline src/objects/object.h:240]: It's still there, just a few lines below?

Comment (matthiasbeyer) [inline src/objects/object.h:240]: But the `@warning` tag is removed from it. It is not a warning anymore, but normal documentation. This shouldn't be.

Comment (xVin) [inline src/objects/object.h:240]: Ah, sorry. Didn't know that a newline nulls the tag. Gonna fix asap.

**Comment by neithernut** [comment]
ACK.

**Comment by matthiasbeyer** [comment]
@xVin Please squash your commits into one commit.

**Comment by matthiasbeyer** [comment]
ACK. Merged.

**Pull request merged**